package.json

```diff
@@ -23,7 +23,8 @@
     "gulp-jsdoc-to-markdown": "^1.1.1",
     "gulp-mocha": "^2.1.3",
     "gulp-rename": "^1.2.2",
-    "promise": "^7.0.4",
+    "path-exists": "^3.0.0",
+    "pinkie": "^2.0.4",
```

Comment:
AFAIK just setting `ecmaVersion` to `5` should be enough (ESLint / Espree uses Acorn under the hood, so when it passes `ecmaVersion: 5`, Acorn simply won't recognize ES6 syntax and won't be able to parse it).